{"name": "test_demo", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 4", "trace": "@pytest.mark.flaky(reruns=2,reruns_delay=2)\n    def test_demo():\n>       assert 3==4\nE       assert 3 == 4\n\ntest_pytest-allure-report.py:37: AssertionError"}, "start": 1641970758345, "stop": 1641970758345, "uuid": "3f7f1ed9-cb74-481f-9062-1f03b6522e1c", "historyId": "7114efc4a77413aa08cd40255d817ae3", "testCaseId": "6b75df94154985552f2023c55dad89d1", "fullName": "test_pytest-allure-report#test_demo", "labels": [{"name": "tag", "value": "flaky(reruns=2, reruns_delay=2)"}, {"name": "suite", "value": "test_pytest-allure-report"}, {"name": "host", "value": "liyinchideMacBook-Pro.local"}, {"name": "thread", "value": "89996-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_pytest-allure-report"}]}